pkg/buildah/inspect.go

```diff
@@ -250,16 +252,42 @@ func inspectImage(ctx context.Context, sc *types.SystemContext, store storage.St
 	if st, ok := transport.(imagestorage.StoreTransport); ok {
 		st.SetStore(store)
 	}
-	ref, err := transport.ParseReference(imgName)
+
+	// try out the candidates as resolved by shortnames. This takes
+	// "localhost/" prefixed images into account as well.
+	candidates, err := shortnames.ResolveLocally(sc, imgName)
 	if err != nil {
-		return nil, nil, err
+		return nil, nil, fmt.Errorf("error resolve shortname %s: %w", imgName, storage.ErrImageUnknown)
+	}
+	// also try out the "docker.io/" prefixed image
```

Comment:
I agree with the `ResolveLocally` functional, but seems we do not need to auto parse imgName for `docker.io` with two reasons below:
1. If the image belongs to `quay.io` or `ghcr.io`
2. if user is using one mirror registry like `dockerproxy.com/docker.io/`

Reply:
Thanks for your review! This function `ParseDockerRef()` is just for backward compatibility. the function returns digested reference, e.g. `docker.io/library/busybox:latest@sha256:xxxx` will be returned as `docker.io/library/busybox@sha256:xxxx`.

Indeed, I guess we didn't have this scenario, so I have removed this function.